### ISSUE TITLE
Error when creating new logic app project. The system cannot find the file specified

### ISSUE THREAD

**Issue by desmondpp** [Azure/logicapps#211]
Does this occur consistently? Yes
Repro steps:

1. Create New Project
2. Choose either stateful or stateless
3. Give workflow a name and hit 'enter'. Error occurs immediately.

Action: azureLogicAppsV2.createNewProject
Error type: Error
Error Message: Command "dotnet "c:\Users\Pat\.vscode\extensions\ms-azuretools.vscode-azurelogicapps-0.0.5\resources\dotnetJsonCli\netcoreapp3.0\Microsoft.TemplateEngine.JsonCli.dll" --require "c:\Users\Pat\AppData\Roaming\Code\User\globalStorage\ms-azuretools.vscode-azurelogicapps\dotnetTemplates\itemTemplates-~3.nupkg" --require "c:\Users\Pat\AppData\Roaming\Code\User\globalStorage\ms-azuretools.vscode-azurelogicapps\dotnetTemplates\projectTemplates-~3.nupkg" --operation create --identity Microsoft.AzureFunctions.ProjectTemplate.CSharp.3.x --arg:name "outomes-forms-logic-apps" --arg:AzureFunctionsVersion v3" failed with exit code "2147614729":
Error: Could not install "c:\Users\Pat\AppData\Roaming\Code\User\globalStorage\ms-azuretools.vscode-azurelogicapps\dotnetTemplates\itemTemplates-~3.nupkg".
Error: Could not load file or assembly 'Microsoft.TemplateEngine.Edge, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.
Error: Could not install "c:\Users\Pat\AppData\Roaming\Code\User\globalStorage\ms-azuretools.vscode-azurelogicapps\dotnetTemplates\projectTemplates-~3.nupkg".
Error: Could not load file or assembly 'Microsoft.TemplateEngine.Edge, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.
{
  "Message": "Could not load file or assembly 'Microsoft.TemplateEngine.Orchestrator.RunnableProjects, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.",
  "Status": -2147352567,
  "TemplateFullName": "Azure Functions",
  "ResultInfo": null,
  "OutputBaseDirectory": null,
  "CreationEffects": null
}

Version: 0.0.5
OS: win32
OS Release: 10.0.18363
Product: Visual Studio Code
Product Version: 1.51.1
Language: en

<details>
<summary>Call Stack</summary>

```
Object.<anonymous> extension.bundle.js:1:136264
fulfilled extension.bundle.js:1:135087
processTicksAndRejections task_queues.js:94:5
```

</details>

**Comment by vikash-anand** [comment]
@desmondpp  We have released a new version of the extension - v0.0.6 today which should fix this issue. Please update your extension and let us know if the issue exists.

**Comment by nidubey1** [comment]
@desmondpp I have tested the same and it works fine now. Reference Document: https://docs.microsoft.com/en-us/azure/logic-apps/create-single-tenant-workflows-visual-studio-code
